db/R/dbfiles.R

```diff
@@ -431,27 +438,29 @@ dbfile.file <- function(type, id, con, hostname=fqdn()) {
 ##' \dontrun{
 ##'   dbfile.id('Model', '/usr/local/bin/sipnet', dbcon)
 ##' }
-dbfile.id <- function(type, file, con, hostname=fqdn()) {
-  if (hostname == "localhost") hostname <- fqdn();
-
+dbfile.id <- function(type, file, con, hostname = fqdn()) {
+  if (hostname == "localhost") 
+    hostname <- fqdn()
+  
   # find appropriate host
-  hostid <- db.query(paste0("SELECT id FROM machines WHERE hostname='", hostname, "'"), con)[['id']]
+  hostid <- db.query(paste0("SELECT id FROM machines WHERE hostname='", hostname, "'"), con)[["id"]]
   if (is.null(hostid)) {
-    invisible(NA)
+    return(invisible(NA))
```

Comment:
`invisible` doesn't by itself return from a function, which is what the intention seemed to be. Changed here and in a number of other places in this file.